docs/source/markdown/podman-image.1.md

```diff
@@ -27,6 +27,7 @@ The image command allows you to manage images
 | push     | [podman-push(1)](podman-push.1.md)                  | Push an image from local storage to elsewhere.                              |
 | rm       | [podman-rmi(1)](podman-rmi.1.md)                    | Removes one or more locally stored images.                                  |
 | save     | [podman-save(1)](podman-save.1.md)                  | Save an image to docker-archive or oci.                                     |
+| scp     | [podman-image-scp(1)](podman-image-scp.1.md)         | Securely copy an image from one host to another.                            |
```

Comment:
```suggestion
| scp      | [podman-image-scp(1)](podman-image-scp.1.md)        | Securely copy an image from one host to another.                            |
```

Reply:
This still looks misaligned.

Reply:
A lot of the lines seem to be misaligned in the file but not in the actual markdown